### DISCUSSION TITLE
Adding Seasons to my DB

### DISCUSSION THREAD

**Discussion by celymyst** [prisma/prisma#16821] [Q&A] [answered]
I want to add Seasons to my db, without using a json[] to store the id and type.
Is that feasible?
to explain it a bit further
lets say i have a series called `The Series` and it has 2 sequels. Then its `relation` field should look like this when requesting the data:
```json
"relations": [
    {
        "id": "uuid of the second season here",
        "type": "SEQUEL"
    },
    {
        "id": "uuid of the third season here",
        "type": "SEQUEL"
    }
]
```
and, by that logic, the `relation` field of the second season would look like this:
```json
"relations": [
    {
        "id": "uuid of the first season here",
        "type": "PREQUEL"
    },
    {
        "id": "uuid of the third season here",
        "type": "SEQUEL"
    }
]
```

**Comment by nurul3101** [comment, marked as answer]
Hello @5H4D0WILA 👋 
Would storing the Sequels in a separate table work for you?
The relation between Series and Sequels could be one-to-many.

You can model the schema like this:
```prisma
model Series {
  id     Int      @id @default(autoincrement())
  title  String
  sequel Sequel[]
}

model Sequel {
  id       Int    @id @default(autoincrement())
  title    String
  series   Series @relation(fields: [seriesId], references: [id])
  seriesId Int
}
```

And Query Like this:
```typescript
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  await prisma.series.create({
    data: {
      title: 'test',
      sequel: {
        createMany: {
          data: [
            {
              title: 'Sequel 1',
            },
            {
              title: 'Sequel 2',
            },
          ],
        },
      },
    },
  });

  const series = await prisma.series.findMany({
    include: {
      sequel: true,
    },
  });

  console.log('series', JSON.stringify(series, null, 2));
}

main()
  .catch((e) => {
    throw e;
  })
  .finally(async () => {
    await prisma.$disconnect();
  });
```

Here's the response:
<img width="571" alt="Screenshot 2022-12-15 at 1 22 02 PM" src="https://user-images.githubusercontent.com/13049676/207803243-71719098-b4dd-40f3-9f69-fdaf92fdb15b.png">

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
